doc-tool/test/JavaConverterTest.scala

```diff
@@ -0,0 +1,111 @@
+package dotty.tools
+package dottydoc
+
+import org.junit.Test
+import org.junit.Assert._
+import model.{
+  Entity,
+  Members,
+  SuperTypes,
+  Modifiers,
+  TypeParams,
+  Constructors => MConstructors,
+  Companion,
+  ReturnValue,
+  ImplicitlyAddedEntity,
+  TypeAlias,
+  Trait,
+  Def,
+  NonEntity
+}
+import model.references.{ConstantReference, TypeReference, NoLink}
+import model.comment.Comment
+import dotty.tools.dotc.core.Symbols.NoSymbol
+import _root_.java.util.{Optional => JOptional, Map => JMap}
+
+class JavaConverterTest {
+  import model.JavaConverters._
+
+  @Test def entityConversions = {
+    trait TestEntity extends Entity {
```

Comment:
No need to create all these intermediate subclasses of `entities.scala` - see below